src/java.desktop/share/classes/javax/swing/plaf/synth/SynthComboBoxUI.java

```diff
@@ -730,9 +753,9 @@ private EditorFocusHandler(JComboBox<?> comboBox) {
             comboBox.addPropertyChangeListener("editor",this);
         }
 
-        public void unregister(){
-            comboBox.removePropertyChangeListener(this);
-            if (editorComponent!=null){
+        public void unregister() {
+            comboBox.removePropertyChangeListener("editor", this);
```

Comment:
The listener is added as "comboBox.addPropertyChangeListener("editor",this);" and should be removed by the pair API.